eslint-plugin-expensify/tests/prefer-at.test.js

```diff
@@ -0,0 +1,69 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../prefer-at');
+const message = require('../CONST').MESSAGE.PREFER_AT;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('prefer-at', rule, {
+    valid: [
+        {
+            code: 'example.at(0)',
+        },
+        {
+            code: 'test.at(1)',
+        },
+        {
+            code: 'sample.at(-1)',
+        },
+        {
+            code: 'testing.at(-2)',
+        },
+        {
+            code: 'testing.at(0).at(-1)',
+        },
+        {
+            code: 'test.at((test.length - 1) / 2)',
+        },
+    ],
+    invalid: [
```

Comment:
For each invalid test case here, we must assert we are dealing with arrays and not objects